R/backend-postgres.R

```diff
@@ -235,6 +235,29 @@ sql_translation.PqConnection <- function(con) {
         )
         sql_expr(DATE_TRUNC(!!unit, !!x))
       },
+
+      # clock ---------------------------------------------------------------
+      add_days = function(x, n, ...) {
+        check_dots_empty()
+        sql_expr((!!x + !!n%*INTERVAL%'1 day'))
+      },
+      add_years = function(x, n, ...) {
+        check_dots_empty()
+        sql_expr((!!x + !!n%*INTERVAL%'1 year'))
```

Comment:
@mgirlich this works but is very awkward. Is there a better way to get the correct SQL here?

This is what I want to write but it is not a valid R expression
`sql_expr((!!x + !!n * INTERVAL'1 year'))`

The SQL should look like `\`column_name\` + 2 * INTERVAL'1 year'`

Reply:
@hadley is there a better way to write this expression? `!!x + !!n%*INTERVAL%'1 year'` seems strange but it's the only way I could come up with that worked.

Reply:
Ok I tried glue_sql2 instead. Hopefully I used it correctly.